build/workbox.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs')
+const swBuild = require('workbox-build')
+const config = require('./workbox.config.json')
+
+const updateUrl = (manifestEntries) => manifestEntries.map((entry) => {
+  if (entry.url.startsWith('_gh_pages/')) {
```

Comment:
One minor change here, maybe move `_gh_pages` to a `const` instead of hardcoding it in this file.

Reply:
Just updated this